News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListActivity.java

```diff
@@ -336,6 +337,17 @@ public void onConfigurationChanged(@NonNull Configuration newConfig) {
 		}
 	}
 
+	void showChangelogIfNecessary() {
+		// on first app start with new version - always show the changelog
+		int currentVersionCode = BuildConfig.VERSION_CODE;
+		int previousVersionCode = mPrefs.getInt("PREVIOUS_VERSION_CODE", 0);
```

Comment:
Put the key into a static final String variabel to avoid typos and magic Strings

Reply:
Good idea 👍  fixed in [use constant](https://github.com/nextcloud/news-android/pull/1286/commits/ba5896d92b18db4aea5dedb62662753a7e143f46)